.github/workflows/ci-mobile-save-for-later.yaml

```diff
@@ -0,0 +1,44 @@
+name: CI
+on: push
+
+jobs:
+  mobile-save-for-later-ci:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v3
+      - uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+          aws-region: eu-west-1
+      - name: Set up JDK 8
+        uses: actions/setup-java@v3
+        with:
+          java-version: 8
+          distribution: temurin
+          cache: sbt
+
+      - name: Create artifacts
+        run: sbt "project mobile-save-for-later" assembly
+
+      - name: Build CDK
```

Comment:
I notice that the equivalent `ci-mobile-save-for-later-user-deletion.yaml` file doesn't have this section. Just for my own understanding, why is this needed for mobile S4L but not for the user deletion?